week2/homework/src/index.js

```diff
@@ -1,3 +1,112 @@
 'use strict';
-
 // TODO: Write the homework code in this file
+
+const fs = require("fs");
+
+const help = [
+  "Run 'list' to show the current To-Do list, the item must be inside quotes ' '.",
+  "Run 'add' and then the Item you want to add to a To-Do list, the item must be inside quotes ' '.",
+  "Run 'remove' and then the Item then the item's Number to remove item from To-Do list, the item must be inside quotes ' '.",
+  "Run 'reset' to remove all To-Do items from the list, the item must be inside quotes ' '.",
+  "Run 'update' and then the item's Number you want to update in To-Do list then New Text, the item must be inside quotes ' '."
+];
+
+let command = process.argv[2];
+let index = process.argv[3];
+let item = process.argv[4];
+let argvLen = process.argv.length;
+const file = fs.readFileSync("./to-dos.json", "utf8");
+let toDos = JSON.parse(file);
+
+function helpMe() {
+  help.forEach(el => console.log(el));
+}
+
+function showList() {
+  if (toDos[0]) {
+    toDos.forEach((el, i) => console.log(`${i + 1}- ${el.task}`));
+  } else {
+    console.log("the to-Do list is empty");
+  }
+}
+
+function writeData(json) {
+  let data = JSON.stringify(json, null, 2);
+  fs.writeFile("to-dos.json", data, (err) => {
+    if (err) {
+      console.log("There is something wrong happened, try again!");
```

Comment:
This is an error, right? why don't use `console.error`? also, it's a good practice in this case to include the error itself so it's easier to understand what's wrong.
```js
console.error('Something wrong happened, try again!', error);
```